packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -478,6 +504,76 @@ class AndroidCameraCameraX extends CameraPlatform {
         exposureState.exposureCompensationStep;
   }
 
+  /// Sets the focus mode for taking pictures.
+  ///
+  /// Setting [FocusMode.locked] will lock current focus point if one exists or
+  /// center of entire sensor area if not and will stay locked until either:
+  ///   * Another focus point is set via [setFocusPoint] (which will then become
+  ///     the locked focus point), or
+  ///   * Locked focus mode is unset by setting [FocusMode.auto].
+  @override
+  Future<void> setFocusMode(int cameraId, FocusMode mode) async {
+    switch (mode) {
+      case FocusMode.auto:
+        if (_currentFocusMode == FocusMode.auto) {
+          // CameraX uses auto-focus mode by default, so no need to reconfigure
+          // auto-focus if already set.
+          return;
+        }
+
+        // Determine auto-focus point to restore, if any. We do not restore
+        // default auto-focus point if set previously to lock focus.
+        final MeteringPoint? autoAfPoint = _defaultFocusPointLocked
+            ? null
+            : currentFocusMeteringAction!.meteringPointInfos
+                .where(((MeteringPoint, int?) meteringPointInfo) =>
+                    meteringPointInfo.$2 == FocusMeteringAction.flagAf)
+                .toList()
+                .first
+                .$1;
+        _defaultFocusPointLocked = false;
+
+        await _startFocusAndMeteringFor(
+            meteringPoint: autoAfPoint,
+            meteringMode: FocusMeteringAction.flagAf);
+      case FocusMode.locked:
+        MeteringPoint? lockedAfPoint;
+
+        // Determine if there is an auto-focus point set currently to lock.
+        if (currentFocusMeteringAction != null) {
+          final List<(MeteringPoint, int?)> possibleCurrentAfPoints =
+              currentFocusMeteringAction!.meteringPointInfos
+                  .where(((MeteringPoint, int?) meteringPointInfo) =>
+                      meteringPointInfo.$2 == FocusMeteringAction.flagAf)
+                  .toList();
+          lockedAfPoint = possibleCurrentAfPoints.isEmpty
+              ? null
+              : possibleCurrentAfPoints.first.$1;
+        }
+
+        // If there isn't, lock center of entire sensor area by default.
+        if (lockedAfPoint == null) {
+          lockedAfPoint = proxy.createMeteringPoint(0.5, 0.5, 1, cameraInfo!);
```

Comment:
This point will center focus on entire sensor area and was recommended to me to use in this case.